WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -524,8 +525,10 @@ - (void)updateCommentForNewContent:(NSString *)newContent
 - (void)editReply
 {
     __typeof(self) __weak weakSelf = self;
-
-    EditReplyViewController *editViewController = [EditReplyViewController newEditViewController];
+    
+    BOOL shouldAddSuggestionView = [[SuggestionService sharedInstance] shouldShowSuggestionsForSiteID:self.comment.blog.blogID];
+    NSNumber *siteID = shouldAddSuggestionView ? self.comment.blog.blogID : nil;
+    EditReplyViewController *editViewController     = [EditReplyViewController newReplyViewControllerForSiteID:siteID];
```

Comment:
Annoying comment here: It'll probably be more explicit to just send the blogID, and let the `EditReplyViewController` determine if it should show suggestions or not.

(Or send a boolean saying... "shouldShowSuggestions", but since now you've encapsulated all that logic in the SuggestionsService, it'd be okay to let it determine what to do).